public_html/wp-content/mu-plugins/blocks/blocks.php

```diff
@@ -14,13 +14,17 @@
  */
 function load() {
 	if ( ! function_exists( 'register_block_type' ) ) {
+		// we can remove this now
 		return;
 	}
 
 	require_once PLUGIN_DIR . 'includes/speakers.php';
 }
 
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load' );
+// what do you think about moving all of the add_action|filter to the top of the file?
+// i personally like that because it serves as a "table of contents" for what's in the file
+// and then the rest of the file just has function declerations, rather than a mix of declerations and executing code
```

Comment:
Yeah, I think you're right that duplicating the info in a comment would lead to it getting out of sync. Sounds like it's just personal preference then, so i'm fine leaving it like this.